databroker/src/grpc/kuksa_val_v2/conversions.rs

```diff
@@ -215,94 +217,170 @@ impl From<&proto::Datapoint> for broker::DataValue {
-                            r#type: Some(proto::value_restriction::Type::Unsigned(
-                                proto::ValueRestrictionUint {
-                                    allowed_values: vec.iter().cloned().map(u64::from).collect(),
-                                    min: None, // TODO: Implement
-                                    max: None, // TODO: Implement
-                                },
-                            )),
-                        })
-                    }
-                    broker::DataValue::Uint64Array(vec) => {
-                        Some(proto::ValueRestriction {
-                            r#type: Some(proto::value_restriction::Type::Unsigned(
-                                proto::ValueRestrictionUint {
-                                    allowed_values: vec.clone(),
-                                    min: None, // TODO: Implement
-                                    max: None, // TODO: Implement
-                                },
-                            )),
-                        })
-                    }
-                    broker::DataValue::FloatArray(vec) => {
-                        Some(proto::ValueRestriction {
-                            r#type: Some(proto::value_restriction::Type::FloatingPoint(
-                                proto::ValueRestrictionFloat {
-                                    allowed_values: vec.iter().cloned().map(f64::from).collect(),
-                                    min: None, // TODO: Implement
-                                    max: None, // TODO: Implement
-                                },
-                            )),
-                        })
-                    }
-                    broker::DataValue::DoubleArray(vec) => {
-                        Some(proto::ValueRestriction {
-                            r#type: Some(proto::value_restriction::Type::FloatingPoint(
-                                proto::ValueRestrictionFloat {
-                                    allowed_values: vec.clone(),
-                                    min: None, // TODO: Implement
-                                    max: None, // TODO: Implement
-                                },
-                            )),
-                        })
-                    }
+                    });
+                };
+            }
+            DataType::Int8
+            | DataType::Int16
+            | DataType::Int32
+            | DataType::Int64
+            | DataType::Int8Array
+            | DataType::Int16Array
+            | DataType::Int32Array
+            | DataType::Int64Array => {
+                let min_value = match metadata.min {
+                    Some(DataValue::Int32(value)) => Some(i64::from(value)),
+                    Some(DataValue::Int64(value)) => Some(value),
+                    // Assumption here that we already have checked types
+                    // so wither Int64 or None
```

Comment:
Can you update the wording of all of these comments a bit?

"so wither Int64 or None" isn't that understandable. Also copy / pasted below a few times.

Reply:
Yep, it is coming. Intention is to show that it never shall be anything else than Int32/Int64/None, so safe to assume that `_` never will be anything else than `None`.